src/sage/rings/finite_rings/finite_field_base.pyx

```diff
@@ -1229,6 +1230,37 @@ cdef class FiniteField(Field):
             True
             sage: all(to_V(h(c) * e) == c * to_V(e) for e in E for c in F)
             True
+
+        TESTS::
+
+            The various calling conventions should not conflict with each other.
```

Comment:
```suggestion
        TESTS:

        The various calling conventions should not conflict with each other::
```

Reply:
Cancelled these changes after reverting the caching patch.